optimism/src/lookup.rs

```diff
@@ -114,3 +114,27 @@ pub trait Lookups {
     /// Adds all lookups of Self to the environment
     fn lookups(&mut self);
 }
+
+/// A table of values that can be used for a lookup, along with the ID for the table.
+#[derive(Debug, Clone)]
+pub struct LookupTable<F> {
```

Comment:
This coincides with the `LookupTable` from kimchi... I'm afraid it can create quite a bit of confusion if one looks at both codebases in a comparative way. But it's not a bad name, I guess this struct is the most similar one to the kimchi's `LookupTable` struct anyway.

Reply:
The difference is that in Kimchi the table is transposed. Meaning, the first index refers to the number of columns, whereas here it is the row.